src/utils/beneficial.owner.type.ts

```diff
@@ -0,0 +1,56 @@
+import { NextFunction, Request, Response } from "express";
+
+import { getApplicationData } from "../utils/application.data";
+import { ApplicationData } from "../model";
+import { logger } from "../utils/logger";
+import { checkEntityHasTrusts } from "../utils/trusts";
+import * as config from "../config";
+import { isActiveFeature } from '../utils/feature.flag';
+
+export const getBeneficialOwnerType = (req: Request, res: Response, next: NextFunction, registrationFlag: boolean) => {
```

Comment:
If we have two private functions to do the work then should this just be two controllers?

Reply:
can't think of why I separated them but makes sense for them to be separate